model/Hardware/Classes/HBOMDimensions.md

```diff
@@ -0,0 +1,39 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# HBOMDimensions
```

Comment:
Why is this a separate class instead of these being properties of HBOMPhysical ?
And what does HBOM mean, anyway?